### ISSUE TITLE
Ensure that child.deleted is appropriately used throughout the codebase

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#614]
**TL;DR**: Double check that the child.deleted property is actually used to restrict child data shown/available as appropriate.

**Acceptance Criteria**
A precise description of how/when this issue will be considered closed, e.g.:

- [x] Deleted child not displayed in child registration list, & test.
- [x] Deleted child not displayed in participant view, & test.
- [x] Deleted child still available in child data download, & test. 
- [x] Deleted child not included in announcement emails, & test.
- [x] Deleted child not included in summary of all children at bottom of recruitment stats view, & test.
- [x] Deleted child videos still available to participant under past sessions.
